android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandler.kt

```diff
@@ -0,0 +1,63 @@
+package io.logto.sdk.android.auth.logto
+
+import android.app.Activity
+import android.webkit.JavascriptInterface
+import android.webkit.WebView
+import io.logto.sdk.android.auth.social.alipay.AlipaySocialSession
+import io.logto.sdk.android.auth.social.web.WebSocialSession
+import io.logto.sdk.android.auth.social.wechat.WechatSocialSession
+
+class LogtoWebViewSocialHandler(
+    private val webView: WebView,
+    private val hostActivity: Activity,
+) {
+    companion object {
+        const val SOCIAL_HANDLER_NAME = "SocialHandler"
+    }
+
+    @JavascriptInterface
+    fun postSocialMessage(socialType: String) {
+        // TODO - Conventions Between UI and Native
+        webView.post {
+            handleSocial(socialType)
+        }
+    }
+
+    private fun handleSocial(socialType: String) {
+        println("Social == Social Sign In Click")
+        println("Social == SocialType: $socialType")
+        // TODO - Create Social Sessions Dynamically
```

Comment:
what's this exactly? should we refer to a linear issue to provide more context?

Reply:
LOG-2124

Reply:
done.